Resources/Locale/en-US/reagents/meta/physical-desc.ftl

```diff
@@ -47,15 +47,15 @@ reagent-physical-desc-reasonably-metallic = reasonably metallic
 reagent-physical-desc-metallic = metallic
 reagent-physical-desc-gaseous = gaseous
 reagent-physical-desc-ground-brass = ground brass
-reagent-physical-desc-dark--brown = dark brown
+reagent-physical-desc-dark-brown = dark brown
 reagent-physical-desc-crystalline = crystalline
 reagent-physical-desc-viscous = viscous
 reagent-physical-desc-shiny = shiny
 reagent-physical-desc-dark-red = dark-red
 reagent-physical-desc-ionizing = ionizing
 reagent-physical-desc-nondescript = nondescript
 reagent-physical-desc-burning = burning
-reagent-physical-desc-powdery = powdery
```

Comment:
This will break things that actually are using it, and there's no reason to rename it.

Reply:
I think they tried to fix ashes reagent that incorrectly uses reagent-physical-desc-powder, the real fix is making the ash reagent use reagent-physical-desc-powdery since its more used everywhere

Reply:
That is true, i forgor about other reagents. Give me a minute

Reply:
Ah, I see